packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.js

```diff
@@ -73,7 +75,7 @@ function Unstable_TrapFocus(props) {
       rootRef.current.focus();
     }
 
-    const contain = () => {
+    const contain = (event) => {
```

Comment:
Is this a native DOM event or a synthetic React event? As long as we're untyped I'd suggest using `*event` for react events and `*nativeEvent` for native DOM events to make it clear what we're dealing with.

Reply:
From past experience, such distinction is also useful for using `event.key` vs `nativeEvent.keyCode` in regard to how React polyfill IE 11.

Reply:
Renamed to `nativeEvent` 👍